keeper/component/executor/genesis.go

```diff
@@ -34,6 +34,10 @@ func (e *Executor) InitGenesis(ctx context.Context, g *executortypes.GenesisStat
 		return core.ErrNilPointer.Wrap("executor genesis")
 	}
 	for _, id := range g.PausedActionIds {
+		if err := id.Validate(); err != nil {
+			return fmt.Errorf("invalid paused action id: %w", err)
```

Comment:
nit: just to be consistent with other errors
```suggestion
			return fmt.Errorf("invalid paused action ID: %w", err)
```